app/src/main/java/com/veephealthoutloud/healthoutloud/ApplicationActivity.java

```diff
@@ -0,0 +1,92 @@
+package com.veephealthoutloud.healthoutloud;
+
+import android.app.Application;
+import android.text.TextUtils;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.Volley;
+
+/**
+ *  Singleton class
+ *  Used to ensure volley connection lasts the lifetime of app
+ *  Sets up RequestQueue
+ */
+
+public class ApplicationActivity extends Application {
```

Comment:
since this doesn't extend the Activity class, might be helpful to change the name to something more specific (without "Activity")